raven/README.md

```diff
@@ -63,6 +71,10 @@ ivo://ivoa.net/std/UMS#login-0.1 = ivo://cadc.nrc.ca/gms
 
 ivo://ivoa.net/std/CDP#delegate-1.0 = ivo://cadc.nrc.ca/cred
 ivo://ivoa.net/std/CDP#proxy-1.0 = ivo://cadc.nrc.ca/cred
+
+org.opencadc.raven.putPreference=@CADC
+@CADC.resourceID=ivo://cadc.nrc.ca/cadc/minoc
+@CADC.namespaces=cadc:IRIS/ cadc:CGPS/
```

Comment:
This example doesn't even conform to the statement above, but will change further when multiple lines are used.

Reply:
No it doesn't match the description, updated to use namespace with multiple lines.